docs/release-guide.md

```diff
@@ -49,3 +51,13 @@ Pull image from the staging registry.
 ```
 docker pull gcr.io/k8s-staging-descheduler/descheduler:v20200206-0.9.0-94-ge2a23f284
 ```
+
+## Helm Chart
+Helm chart releases are managed by a separate set of git tags that are prefixed with `chart-*`. Example git tag name is `chart-v0.18.0`. Released versions of the
+helm charts are stored in the `gh-pages` branch of this repo. The [chart-releaser-action GitHub Action](https://github.com/helm/chart-releaser-action) is setup to
+build and push the helm charts to the `gh-pages` branch when a `chart-*` git tag is created.
+
+1. Merge all helm chart changes into the appropriate release branch(i.e. release-1.18)
```

Comment:
Before doing a release you need to make sure that the `appVersion` in _Chart.yaml_ matches the latest release version (no `v` prefix) and that the `version` in _Chart.yaml_ (chart version) has been incremented.

Reply:
Fixed in 4415954d50082135e48d2e29b2ed66530b319ced.